libs/coin-modules/coin-bitcoin/src/buildTransaction.test.ts

```diff
@@ -0,0 +1,125 @@
+import { BigNumber } from "bignumber.js";
+import { BitcoinAccount, BitcoinResources, NetworkInfoRaw, Transaction, bitcoinPickingStrategy } from "./types";
+import wallet from "./wallet-btc";
+
+import { fromTransactionRaw } from "./transaction";
+import { listCryptoCurrencies } from "@ledgerhq/cryptoassets/currencies";
+import { emptyHistoryCache } from "@ledgerhq/coin-framework/account/index";
+import { buildTransaction } from "./buildTransaction";
+import { FeeNotLoaded } from "@ledgerhq/errors";
+
+const networkInfo: NetworkInfoRaw = {
```

Comment:
Move feeItems outside and import it in the test files to avoid copying and pasting it in each unit test file.